funnel/serializers.py

```diff
@@ -3,6 +3,7 @@
 
 import pydantic
 import xarray as xr
+import xcollection as xc
```

Comment:
How about making `xcollection` a soft dependency by moving the import inside the `def xcollection()` function?

Reply:
How do we deal with calling `xcollection` in this section if we don't import it outside of that function?

```python
@pick_serializer.register(xc.Collection)
def _(obj):
    return registry.serializers.get('xcollection')().name
```

Reply:
Never mind... This won't work due to the `@pick_serializer.register(xc.Collection)` line... Let's keep it as a hard dependency for the time being... We need to add to the `requirements.txt` and the documentation conda env file.

Reply:
Okay! I included it in the `environment.yml` file, and just updated the `requirements.txt`